apps/iotune/iotune.cc

```diff
@@ -319,19 +319,32 @@ class read_request_issuer : public request_issuer {
     }
 };
 
+std::chrono::duration<double>
+warmup_period(std::chrono::duration<double> duration) {
+    auto min_warmup = 3s;
+    auto ten_percent = duration / 10;
+    return min_warmup.count() > ten_percent.count() ? min_warmup : ten_percent;
```

Comment:
No need to compare counts, durations can be compared directly.

Also, std::max().